portfolio/src/main/java/com/google/sps/listener/IdListener.java

```diff
@@ -45,16 +47,15 @@ public void contextInitialized(ServletContextEvent sce) {
         idNum = id;
       }
     }
-
+    
     String id = Integer.toString(idNum);
-    sc.setAttribute("id", id);
-
+    sc.setAttribute(NEXT_AVAILABLE_ID_ATTRIBUTE, id);
   }
 
   @Override
 	public void contextDestroyed(ServletContextEvent sce) {
-		ServletContext sc = sce.getServletContext();
-		sc.removeAttribute("id");
+	  ServletContext sc = sce.getServletContext();
+	  sc.removeAttribute(NEXT_AVAILABLE_ID_ATTRIBUTE);
 	}
```

Comment:
The method definition is indented too far (should be aligned with the `@Override`)